alibi_detect/utils/keops/kernels.py

```diff
@@ -176,3 +205,10 @@ def forward(self, x_proj: LazyTensor, y_proj: LazyTensor, x: Optional[LazyTensor
         if self.kernel_b is not None:
             similarity = (1-self.eps)*similarity + self.eps*self.kernel_b(x, y)
         return similarity
+
+    def get_config(self) -> dict:
+        return self.config.copy()
```

Comment:
No `flavour` here?

Reply:
Bear with me on this one! We have `flavour` in `ModelConfig` etc (see `saving.schemas`) because we use this to decide which `load_model_...` function to use (i.e. `_tf`, `_pt` or `_sk`). I've kept `flavour` decoupled from the detector `backend` here so that we have the option of using different flavour preprocessing to the model backend (e.g. a scikit-learn preprocessor with a tensorflow MMD detector). Plus, we might have a `model` even when there is no backend (e.g. any preprocessing model with KS detector). 

The situation is a little different for kernels. These are only used for detectors which have backends, and it doesn't make sense to decouple their `flavour` from `backend`. Ideally we would not actually have `flavour` at all for kernels (we actually decide which `load_kernel_...` function to use based on the detector `backend`. The only reason we have `flavour` is so that the `coerce_2_tensor` pydantic validator knows what type of `Tensor` to coerce `sigma` to (this happens before we get to to `load_kernel_...`. 

We don't need to do any coercion for `DeepKernel` hence why no `flavour` in `DeepKernelConfig`. I could add it in just for consistency? But it wouldn't actually be used at load time.